ecs-cli/modules/commands/flags/flags.go

```diff
@@ -110,6 +110,9 @@ const (
 	RoleFlag                                = "role"
 	ComposeServiceTimeOutFlag               = "timeout"
 	ForceDeploymentFlag                     = "force-deployment"
+
+	// Registry Creds
+	UpdateExistingSecretsFlag = "update-existing-secrets"
```

Comment:
I feel like "existing" is probably redundant here given "update" commands are usually understood to apply to something that already exists.

Reply:
I was trying to be explicit that we are updating the value of a specified secret vs. updating what secrets are used for the registries (i.e., creating new ones and swapping them out).

Reply:
If the consensus is that this is obvious then I'll change it.